kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/configuration/GwtNetInterfaceConfigBuilder.java

```diff
@@ -118,6 +119,42 @@ private void setIpv4Properties() {
         this.gwtConfig.setDnsServers(this.properties.getIp4DnsServers(this.ifName));
     }
 
+    private void setIpv6Properties() {
+        Optional<String> status = this.properties.getIp6Status(this.ifName);
+        if (status.isPresent()) {
+            this.gwtConfig.setIpv6Status(status.get());
+        }
```

Comment:
Keep in mind that if the `ip6.status` is missing we have a spacial way to handle it. See:

https://github.com/eclipse/kura/blob/8566aa9cd5feaf9165d6e6d2e352a1b0f491f6e0/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java#L389-L393

We might want to replicate the logic on the UI side to represent it correctly to the user.

Reply:
Maybe it's better to implement those checks on the `TabIp6Ui`, so that the adapter code does not need to perform any checks. What do you think?

Reply:
Yeah. This is just a representation issue therefore should be handled only by the frontend.